BUILDING_DETAILS.md

```diff
@@ -2,6 +2,20 @@
 
 This document covers details that may be useful to some users.
 
+## Building additional yocto packages
+
+Assuming the package you want to add is already present in one of the layers
+configured in [`bisdn-linux.yaml`](bisdn-linux.yaml), you can add it to the
+BISDN Linux image by adding a `IMAGE_INSTALL:append` in `bisdn-linux.yaml`
+(the file contains a commented out example).
```

Comment:
How about providing example .yaml file contents that appends a package and adds a currently unused layer? Then users don't need to modify any tracked files.

And then tell them to use `bisdn-linux.yaml:custom-changes.yaml`. That would at least feel a bit more kas-y to me.

Reply:
Not having to change tracked files is definitely a good idea. I need to check for a suitable repo/layer/recipe if we want to add a repo, too.

Reply:
We can also leave out the "adding a layer" part at first, since adding layers may have unexpected consequences if they contain `.bbappend`s for used recipes, so this may be its own can of worms.